### PR TITLE
refactor: improvements and fixes to ButtonLink and SiteNotificationAlert

### PR DESCRIPTION
**Pull request by sourcefilter**
QA issues and styling, plus a refactor of css transitions to animate the clip-path property

### CODE REVIEW

**Comment by sourcefilter** [comment]
https://deploy-preview-277--ucla-library-storybook.netlify.app/?path=/story/button-link--with-controls
https://deploy-preview-277--ucla-library-storybook.netlify.app/?path=/story/global-sitenotificationalert--default

File: src/lib-components/SiteNotificationAlert.vue [hunk 1]
```diff
@@ -92,93 +93,89 @@ export default {
...
-        font-size: 16px;
-        font-weight: 400;
-        letter-spacing: 0.01em;
-        line-height: 22px;
-        max-height: 128px;
-        overflow-y: auto;
-        padding: 10px 32px 8px 32px;
+        .button-dismiss {
+            margin: 0 24px 24px;
+            padding: 13px 32px;
```
Comment (dianneweinthal) [inline src/lib-components/SiteNotificationAlert.vue:158]: can remove padding and delete this line

**Review by pghorpade — APPROVED** [review]
👍

**Pull request merged**